src/components/app/routes/queries/subsidies/subscriptions.js

```diff
@@ -12,29 +12,29 @@ async function fetchSubscriptionLicensesForUser(enterpriseUUID) {
   return camelCaseObject(response.data);
 }
 
-async function fetchCustomerAgreementData(enterpriseUUID) {
-  const queryParams = new URLSearchParams({
-    enterprise_customer_uuid: enterpriseUUID,
-  });
-  const config = getConfig();
-  const url = `${config.LICENSE_MANAGER_URL}/api/v1/customer-agreement/?${queryParams.toString()}`;
-  const response = await getAuthenticatedHttpClient().get(url);
-  return camelCaseObject(response.data);
-}
-
 /**
  * TODO
  * @param {*} param0
  * @returns
  */
 async function fetchSubscriptions(enterpriseUuid) {
   const results = await Promise.all([
-    fetchCustomerAgreementData(enterpriseUuid),
     fetchSubscriptionLicensesForUser(enterpriseUuid),
   ]);
+  camelCaseObject(results);
+
+  // Refactor response to match current state of the world before
+  // reintegrating components
+  const customerAgreement = {
+    results: [results[0].results[0].customerAgreement],
```

Comment:
There is too much `results` in here for me to understand how this works. :)
Can you rename the first `results` like `responseData` or something?  Would that work?  Also, isn't the thing returned by the fetch of subscription data an object with a paginated `results` list inside it?

Reply:
Addressed with `response`

Reply:
Updated the response objects to take pagination into account as well, and with cases of multiple subscriptions. 
![Screenshot 2024-02-26 at 11 58 55 AM](https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/82611798/9138f90e-2dd6-4872-978f-04c15e814664)

Reply:
nice!